app/models/report.rb

```diff
@@ -1,4 +1,9 @@
 class Report < ActiveRecord::Base
+  STATUS_CREATED = 10
```

Comment:
What is the gain of obfuscating status with some codes? Could we simply enumerate the status and store them as-is in database?